Automattic/jetpack
# Fix paid requests localization

Fixes Automattic/i18n-issues#853

## Proposed changes:

* Updates the Jetpack Search features table to use the user's locale for number formatting.
* This change replaces hardcoded 'en-US' locales with the user's locale for better internationalization support.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Check out the branch with the changes.
2. Set the user's locale to Japanese.
3. Navigate to the Jetpack Search screen (`/wp-admin/admin.php?page=jetpack-search`) and view the Features table.
4. Verify that the number of records and requests is correctly formatted based on the user's locale. Specifically, 1万 is present.
